examples/pcap_example.rs

```diff
@@ -32,7 +32,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         _ => todo!(),
     };
 
+    let mut counter = 0;
     while let Ok(packet) = cap.next_packet() {
+        match opts.num_packets {
```

Comment:
You can simply do `if opts.num_packets.is_some() && ...` . That is slightly more readable.

Reply:
Do you mean something like what I did now?